packages/use-i18n/src/__tests__/usei18n.tsx

```diff
@@ -111,113 +111,6 @@ describe('i18n hook', () => {
     })
   })
 
-  it('should use specific load on useTranslation', async () => {
```

Comment:
why did you removed all test on useTranslation ?

Reply:
Because I removed prefix, but it was a mistake. Just added them back without prefix.